### ISSUE TITLE
heretic auto fly with right joystick issue

### ISSUE THREAD

**Issue by ContraVania** [hh79/gzdoomvr#52]
When I press up on the right joystick on a Valve Index controller, it causes me to use Wings of Wrath and start flying even if I don't have that item selected.

**Comment by hh79** [comment]
Yes, it annoys me to. It will be fixed when I finally do the complete overhaul how controller mappings work.
